.gitignore

```diff
@@ -2,4 +2,5 @@
 *.odp
 *.swp
 *.~lock*
+
```

Comment:
?

Reply:
You can ignore this. I actually committed with some other email in my local github config. I just needed to overwrite the previous commit by my username(email).

Reply:
Generally, don't do anything that will show up in the diff. You could just amend the previous commit without changes, I think. This will make this change show in the diff and have reviewers wonder what you did. Also, now there's a blank line in the ``.gitignore``...

Reply:
Yes, Sorry about that! I will be mindful of that next time.